internal/plugins/golang/plugin.go

```diff
@@ -18,15 +18,15 @@ import (
 	"github.com/operator-framework/operator-sdk/internal/plugins"
 
 	"sigs.k8s.io/kubebuilder/pkg/plugin"
-	kbgov2 "sigs.k8s.io/kubebuilder/pkg/plugin/v2"
+	kbgov3 "sigs.k8s.io/kubebuilder/pkg/plugin/v3"
 )
 
 const (
 	// These will be used as plugin name/version in phase 2 plugins when we can
 	// pipe kubebuilder's init/create api output into our scaffold modification
 	// plugins. In phase 1 we wrap kubebuilder's Go plugin to have the same effect.
 	pluginName    = "go" + plugins.DefaultNameQualifier
-	pluginVersion = "v2.0.0"
+	pluginVersion = "v2.2.2" // todo: ping 3.0.0 -> it is just to ensure that will address fix
```

Comment:
HI @estroz, 
 
Just to ensure that the specific plugin resolution fixes worked as expected and fixed the issue faced here as well where the SDK was not pinning its version. See; 

```
 $ operator-sdk init
Writing scaffold for you to edit...
Get controller runtime:
$ go get sigs.k8s.io/controller-runtime@v0.6.0
Update go.mod:
$ go mod tidy
Running make:
$ make
/Users/camilamacedo/go/bin/controller-gen object:headerFile="hack/boilerplate.go.txt" paths="./..."
go fmt ./...
go vet ./...
go build -o bin/manager main.go
Next: define a resource with:
$ operator-sdk create api
camilamacedo@Camilas-MacBook-Pro ~/go/src/test $ cat PROJECT 
domain: my.domain
layout: go.kubebuilder.io/v3.0.0
repo: test
version: 3-alpha
plugins:
  go.operator-sdk.io/v2.2.2: {}

```